inputs/mvs_config.json

```
@@ -379,93 +335,97 @@
             "outflow_direction": "Electricity (DSO)",
             "peak_demand_pricing": {
                 "unit": "currency/kW",
-                "value": 0
+                "value": 60
```

```suggestion
                "value": 0
```

0 for no peak demand pricing